test/runner/lib/delegation.py

```diff
@@ -154,6 +154,7 @@ def delegate_docker(args, exclude, require):
     util_image = args.docker_util
     test_image = args.docker
     privileged = args.docker_privileged
+    memory_limit = args.docker_memory
```

Comment:
No need to copy this to a local variable.

Yes, the existing code does that for a few args, but it's unnecessary. No need to fix those though.